src/llmq/utils.cpp

```diff
@@ -323,7 +332,7 @@ std::vector<std::vector<CDeterministicMNCPtr>> BuildNewQuorumQuarterMembers(cons
             ss << m->proTxHash.ToString().substr(0, 4) << "|";
         }
         ss << "]";
-        LogPrint(BCLog::LLMQ, "BuildNewQuorumQuarterMembers h[%d] sortedCombinedMnsList[%s]\n",
+        LogPrint(BCLog::LLMQ, "BuildNewQuorumQuarterMembers h[%d] sortedCombinedMns[%s]\n",
```

Comment:
?

Reply:
I just wanted to be aligned with line 426 (so I can search `sortedCombinedMns` and have both of them)

Reply:
https://github.com/dashpay/dash/blob/master/CONTRIBUTING.md#pull-request-philosophy :) but ok